app/lib/core/models/loading_vehicle_info.dart

```diff
@@ -284,6 +317,12 @@ class CompromisedAnimal {
       {@required this.animalDescription,
       @required this.measuresTakenToCareForAnimal});
 
+  factory CompromisedAnimal.defaultCompromisedAnimal(
```

Comment:
- [ ] Remove this constructor. Anyone with a list of these should just have `[]`
> The only thing that needed default constructors is the ATR's immediate fields